_posts/2020-08-06-kubernetes-clients-comparison.md

```diff
@@ -0,0 +1,142 @@
+Most common types are support and common actions are available.
+But Lens has another trick up it's sleeve which makes it different from the other tools: Metrics/Prometheus integration.
+
+The integration relies on a Prometheus instance being installed in the cluster that exposes the supported metrics.
+You can opt for Lens to install Prometheus (and other required components) for you, but in real scenarios you either don't have those rights or you'll already have a Prometheus installed in the cluster.
+In those cases, you can just [add some configuration](https://github.com/lensapp/lens/blob/master/troubleshooting/custom-prometheus.md){:target="_blank" rel="noopener noreferrer"} to that instance and point the Lens app to that Prometheus instance.
+It will use port-forwarding under the hood, so no need to expose the Prometheus instance to the outside.
+The integration works nicely and instantly provides some metrics about your cluster and deployed components.
+This provides good insights for developers to figure out their resource consumption without leaving their Kubernetes client.
+The charts and data seem to be very rudimentary, but improvement are expected to arrive over time.
+
+As most of the client described in this post, Lens app is an open-source project.
+There seems to be a group / company behind the development, but no paid / supported version is available at this time.
+There is very active development happing on the app and releases are about one month apart, so bugs and new features should be available regularly.
+
+Screenshot: List of pods in Lens
+<img alt="Lens pod list" src="{{ '/img/2020-08-06-kubernetes-clients-comparison/lens-pod-list.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 1000px;">
+
+Screenshot: Details about a pod in Lens, including Prometheus supplied metrics
+<img alt="Lens pod details" src="{{ '/img/2020-08-06-kubernetes-clients-comparison/lens-pod-details.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 1000px;">
+
+Screenshot: Overview of a cluster in Lens, including the last error events
+<img alt="Lens cluster overview" src="{{ '/img/2020-08-06-kubernetes-clients-comparison/lens-cluster-overview.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 1000px;">
+
+## Kubenav
+
+## Infra App
+[Infra App](https://infra.app/){:target="_blank" rel="noopener noreferrer"} is a new addition to the list of Kubernetes clients.
+It is made by the people over at Docker Desktop & [Kitematic](https://kitematic.com){:target="_blank" rel="noopener noreferrer"} and is being developed behind closed doors, which has been addressed as unpleasant within the Kubernetes community.
+It provides you with a clean, simplistic user interface which groups everything you need to know about a single resource together.
+Everything speaks for itself and all the information you need is within the reach of a few simple clicks.
```

Comment:
Everything is self explanatory and all the information you need is available within a few simple clicks